cmd/image-builder/README.md

```diff
@@ -202,3 +202,43 @@ This integration allows the `image-builder` binary to gather inputs directly fro
   enhanced workflow experience** comparable to the GitHub workflow approach.
 - **Build Report**: A dedicated flag (`--build-report-path`) has been added to the `image-builder` binary, enabling it to write a build
   report generated by the ADO pipeline to a specified file. This functionality is intended specifically for use in Jenkins.
+
+## Cross-Compiling and Caching for Non Native Architecture Builds
+
+Image Builder uses builder agents with `linux/amd64` native architecture.
+When building images for multiple architectures or building image for non native architecture,
+consider enabling cross-compilation to significantly reduce build times.
+Testing has shown that cross-compilation can speed up the build process by **10x**.
+Reducing build times from 12 minutes to less than 2 minutes in our tests scenario with rather small golang codebase.
+
+### Key Recommendations
+
+- **Cross-Compilation**: If you are building, builder non native architecture images, implement cross-compilation in your Dockerfile, use
```

Comment:
```suggestion
- **Cross-Compilation**: If you are building non-native architecture images, implement cross-compilation in your Dockerfile, use
```

Reply:
The word 'builder' doesn't seem to fit this context. I removed it, but it's possible that part of this sentence that complements the builder idea is missing.